### PR TITLE
 Port documentation from the Wiki: Date Time Field (#327)

### PR DESCRIPTION
**Pull request by krlevkirill**

### CODE REVIEW

File: _docs/datetime-field.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```xml
+<Field name="CreationDate" type="DateTime">
+...
+</Field>
+```
+
+## Supported Field Controls
+
+[DatePicker Field Control](./DatePicker-Field-Control): a simple grid displaying a clickable calendar in month view
```
Comment (herflis) [inline _docs/datetime-field.md]:
```suggestion
[DatePicker Field Control](./datepicker-fieldcontrol): a simple grid displaying a clickable calendar in month view
```

File: _docs/datetime-field.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Configuration
+
+The following properties can be set in the field's configuration:
+
+- **DateTimeMode**: (optional) an enumeration which defines the presentation mode of the stored value: *None*, *Date* and *DateAndTime*. This only controls the behavior of the [DatePicker Field Control](./DatePicker-Field-Control). Default is *None*.
```
Comment (herflis) [inline _docs/datetime-field.md]:
```suggestion
- **DateTimeMode**: (optional) an enumeration which defines the presentation mode of the stored value: *None*, *Date* and *DateAndTime*. This only controls the behavior of the [DatePicker Field Control](./datepicker-fieldcontrol). Default is *None*.
```

File: _docs/datetime-field.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- **Precision**: (optional) an enumeration which defines the precision of the indexed value: *Millisecond*, *Second*, *Minute*, *Hour*, *Day* (Default is *Minute*). This does not affect the stored value, only the value stored in the index, making it possible to use different precision levels depending on the nature of the application. Chosing a finer or coarser precision than the optimal may cause slower query running and larger index files than what would be reasonable. Default is *Minute*.
+
+> For a complete list of common Field Setting configuration properties see [CTD Field definition](./Content-Type-Definition#Field_definition).
```
Comment (herflis) [inline _docs/datetime-field.md]:
```suggestion
> For a complete list of common Field Setting configuration properties see [CTD Field definition](./content-type-definition#Field_definition).
```

**Comment by herflis** [comment]
Hey @krlevkirill ,
thank you for your contribution! 🙏 I've made some minor suggestions, but otherwise its ok. Don't forget to register 👉 [here](https://sensenet.com/Hacktoberfest2019)

File: _docs/datetime-field.md (hunk 1 above)
Comment (krlevkirill) [reply _docs/datetime-field.md]: fixed

File: _docs/datetime-field.md (hunk 2 above)
Comment (krlevkirill) [reply _docs/datetime-field.md]: fixed

File: _docs/datetime-field.md (hunk 3 above)
Comment (krlevkirill) [reply _docs/datetime-field.md]: updated

**Pull request merged**